test/test_graft.py

```diff
@@ -36,6 +36,49 @@
 
 class Tests(unittest.TestCase):
 
+    def test_euk_check(self):
+        reads = """>euk1
+TCAAATGTCTGCCCTATCAACTATTGATGGTAGTGTAGAGGACTACCATGGTTGCGACGGGTAACGGGGAATCAGGGTTCGATTCCGGAGAGGGAGCCTG
+>euk2
+GTTGCGACGGGTAACGGGGAATCAGGGTTCGATTCCGGAGAGGGAGCCTGAGAAATGGCTACCACATCCAAGGAAGGCAGCAGGCGCGTAAATTACCCAA
+>euk3
+AGAAATGGCTACCACATCCAAGGAAGGCAGCAGGCGCGTAAATTACCCAATCCCGGCACGGGGAGGTAGTGACGAGAAATAACAATATGGACCTCTCTAA
+>euk4
+TCCCGGCACGGGGAGGTAGTGACGAGAAATAACAATATGGACCTCTCTAACGATGGTCCATAATTGGAATGAGTTGAGCATAAATCCTTTTGCAAGGATC
+>bac1
+CGGCTTTCATGATGTAACGGGCGGTGTGTACAAGGCCTGGGAACGTATTCACGGCGCCGTAGCTGATGCGCCATTACTAGCGATTCCAACTTCATGTCGT
+>bac2
+GTTGAAACTCAAAGGAATTGACGGGGGCCCGCACAAGCGGTGGAGCATGTGGTTTAATTCGACGCAACGCGAAGAACCTTACCTGGGCTTGACATGGTAG
+>bac3
+CTTGTGCGGGGACCCGTCAATTCCTTTGAGTTTTAACCTTGCGGCCGTAGTCCCCAGGCGGTGAACTTATCGCGTTAGCTTCGGCACCGACGGCTTTGAA
+>bac4
+CCCCGCCTTCCTCCCCTTTGTCAGAGGCAGTTCTGCTAGAGTGCGCCCCGATTGCTCGGGGGTAGCAACTAACCGTAAGGGTTGCGCTCGTTGCGGGACT"""
+        with tempfile.NamedTemporaryFile(suffix='.fa') as fasta:
+            fasta.write(reads)
+            fasta.flush()
+            data = fasta.name
+            package = os.path.join(path_to_data,'61_otus.gpkg')
+
+            with tempdir.TempDir() as tmp:
+                cmd = '%s graft --verbosity 2 --forward %s --graftm_package %s --output_directory %s --force --euk_check' % (path_to_script,
```

Comment:
if you use extern not subprocess then you can set the verbosity to 5 and you'll get a decent error message, but don't worry for now